Assets/MixedRealityToolkit-SDK/Features/Input/FocusProvider.cs

```diff
@@ -232,7 +233,7 @@ private void Awake()
                 CreateUiRaycastCamera();
             }
 
-            foreach (var inputSource in InputSystem.DetectedInputSources)
+            foreach (var inputSource in MixedRealityManager.InputSystem.DetectedInputSources)
```

Comment:
Not checking for no Input system
```
MixedRealityManager.InputSystem?.
```
Update and check throughout

Reply:
Focus provider would never start without it, so checking for null is redundant

Reply:
we now await until input system is valid